build.gradle

```diff
@@ -6,8 +6,9 @@ plugins {
     id 'com.jfrog.artifactory'
     id 'org.javamodularity.moduleplugin'
     id 'com.github.johnrengelman.shadow'
-    id 'biz.aQute.bnd.builder' apply false
     id 'com.dorongold.task-tree'
+    id 'biz.aQute.bnd.builder' apply false
+    id "org.jetbrains.kotlin.jvm" apply false
```

Comment:
Only `quasar-kotlin` project cares about the Kotlin plugin, which is another reason to keep that project's build script separate.

Reply:
OK, NP, I'll put it back separately.